### PR TITLE
Update webclient.md

### PR DESCRIPTION
**Pull request by Pri3st**
Added more ways to enable the webclient service in an unprivileged session.

### CODE REVIEW

**Comment by Pri3st** [comment]
Created a new PR because I had to merge previous commits.

File: docs/src/ad/movement/mitm-and-coerced-authentications/webclient.md [hunk 1]
```diff
@@ -109,11 +109,172 @@ https://whatever/
...
 ## Resources
 
 [https://pentestlab.blog/2021/10/20/lateral-movement-webclient](https://pentestlab.blog/2021/10/20/lateral-movement-webclient)
+[https://www.tiraniddo.dev/2015/03/starting-webclient-service.html](https://www.tiraniddo.dev/2015/03/starting-webclient-service.html)
 
 [https://www.webdavsystem.com/server/access/windows](https://www.webdavsystem.com/server/access/windows)
+
+---
+authors: [Pri3st](https://github.com/Pri3st)
+---
```
Comment (ShutdownRepo) [inline docs/src/ad/movement/mitm-and-coerced-authentications/webclient.md:280]: this needs to be at the top of the file, along other authors

File: docs/src/ad/movement/mitm-and-coerced-authentications/webclient.md [hunk 2]
```diff
@@ -109,11 +109,172 @@ https://whatever/
...
+```
+
+=== SharpStartWebclient
+
+By compiling and executing the [SharpStartWebclient](https://github.com/eversinc33/SharpStartWebclient) tool created by [eversinc33](https://github.com/eversinc33)
+
+=== Beacon Object File
+
+[BOF](https://github.com/outflanknl/C2-Tool-Collection/blob/main/BOF/StartWebClient/SOURCE/StartWebClient.c) created by [outflanknl](https://github.com/outflanknl)
+```
```
Comment (ShutdownRepo) [inline docs/src/ad/movement/mitm-and-coerced-authentications/webclient.md:234]: can you specify it's `C` code here?

Comment (ShutdownRepo) [inline docs/src/ad/movement/mitm-and-coerced-authentications/webclient.md:234]: do you mind putting the code in a collapsible details block?
````md
::: details
```C
...code
```
:::
````

File: docs/src/ad/movement/mitm-and-coerced-authentications/webclient.md [hunk 3]
```diff
@@ -109,11 +109,172 @@ https://whatever/
 
 By opening an interactive session with the target (e.g. RDP), opening the Explorer, and type something in the address bar.
 
+=== C# PoC to enable WebClient Programmatically
+
+According to [tiraniddo's research](https://www.tiraniddo.dev/2015/03/starting-webclient-service.html), the webclient service is registered with a service trigger, meaning it can be started automatically in response to a specific system event.
+A simple way to start the service in an unprivileged session is by compiling and executing the following [C# PoC](https://gist.github.com/klezVirus/af004842a73779e1d03d47e041115797) created by [klezVirus](https://gist.github.com/klezVirus).
+```c#
```
Comment (ShutdownRepo) [inline docs/src/ad/movement/mitm-and-coerced-authentications/webclient.md:116]: do you mind putting the code in a collapsible details block?
````md
::: details
```C
...code
```
:::
````

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]
thank you for the PR, here are some change requests

**Comment by Pri3st** [comment]
All changes are made.

**Comment by ShutdownRepo** [comment]
I simplified the PR by removing the PoCs since you already provided links to them and since they were breaking the tabs (details within tabs don't behave well). If you think we need to include the PoCs, then the best would probably be to offer them as downloadable assets (and I advise to do that in another PR)

**Review by ShutdownRepo — APPROVED** [review]

**Pull request merged**